core/vm/operations_verkle.go

```diff
@@ -81,6 +81,10 @@ func makeCallVariantGasEIP4762(oldCalculator gasFunc) gasFunc {
 		if _, isPrecompile := evm.precompile(target); isPrecompile {
 			return gas, nil
 		}
+		// The charging for the value transfer is done BEFORE subtracting
+		// the 1/64th gas, as this is considered part of the CALL instruction.
+		// But the message call is part of the subcall, for which only 63/64th
+		// of the gas should be available.
```

Comment:
this is the explanation I was missing, so now that it's clarified I think the fix is correct.

Reply:
Yes, I think this clarification is correct but could be a bit confusing since there's no value charging here. I know you're referring to code somewhere else, but other reader maybe be confused? (or maybe not). Just a comment.